core/src/main/java/com/linecorp/armeria/common/AggregatedHttpRequest.java

```diff
@@ -113,26 +113,7 @@ static AggregatedHttpRequest of(HttpMethod method, String path, MediaType mediaT
         requireNonNull(path, "path");
         requireNonNull(mediaType, "mediaType");
         requireNonNull(content, "content");
-        return of(method, path, mediaType, HttpData.of(content));
-    }
-
-    /**
-     * Creates a new HTTP request.
-     *
-     * @param method the HTTP method of the request
-     * @param path the path of the request
-     * @param mediaType the {@link MediaType} of the request content
-     * @param content the content of the request
-     * @param offset the start offset of {@code content}
-     * @param length the length of {@code content}
-     */
-    static AggregatedHttpRequest of(
-            HttpMethod method, String path, MediaType mediaType, byte[] content, int offset, int length) {
-        requireNonNull(method, "method");
-        requireNonNull(path, "path");
-        requireNonNull(mediaType, "mediaType");
-        requireNonNull(content, "content");
-        return of(method, path, mediaType, HttpData.of(content, offset, length));
+        return of(method, path, mediaType, HttpData.wrap(content));
```

Comment:
I'm wondering is this method supposed to wrap or copy or be deleted. The javadoc doesn't mention the potential danger of wrapping like `HttpData` javadoc always has. Should we update the javadoc?

Reply:
That'd be nice. How about mentioning that `byte[]` is wrapped via `HttpData.wrap()`?

Reply:
Thanks!